app/javascript/app/form-field-format.js

```diff
@@ -1,37 +1,49 @@
-import { SocialSecurityNumberFormatter, TextField } from 'field-kit';
-import DateFormatter from './modules/date-formatter';
-import NumericFormatter from './modules/numeric-formatter';
-import PersonalKeyFormatter from './modules/personal-key-formatter';
-import ZipCodeFormatter from './modules/zip-code-formatter';
-
+import Cleave from 'cleave.js';
 
 function formatForm() {
-  const formats = [
-    ['.dob', new DateFormatter()],
-    ['.mfa', new NumericFormatter()],
-    ['.personal-key', new PersonalKeyFormatter()],
-    ['.ssn', new SocialSecurityNumberFormatter()],
-    ['.zipcode', new ZipCodeFormatter()],
-  ];
+  if (document.querySelector('.dob')) {
+    new Cleave('.dob', {
+      date: true,
+      datePattern: ['m', 'd', 'Y'],
+    });
+  }
 
-  formats.forEach(function(f) {
-    const [el, formatter] = f;
-    const input = document.querySelector(el);
-    if (input) {
-      /* eslint-disable no-new, no-shadow */
-      const field = new TextField(input, formatter);
+  if (document.querySelector('.personal-key')) {
+    new Cleave('.personal-key', {
+      blocks: [4, 4, 4, 4],
+      delimiter: '-',
+    });
+  }
 
-      // add date format placeholders only to .dob fields
-      if (el === '.dob') {
-        field.setFocusedPlaceholder('');
-        field.setUnfocusedPlaceholder('mm/dd/yyyy');
-      }
+  if (document.querySelector('.backup_code')) {
```

Comment:
I noticed we didn't have backup code formatting, and it kind of looks like a personal key... so it made sense to me to add it!